src/atoms/Flex/__tests__/Flex.web.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import View from '../../View';
+import Flex from '../index';
+
+describe('Renders <Flex /> correctly', () => {
+  it('renders child of <Flex />(no styles) correctly', () => {
+    const { container } = render(
+      <Flex>
+        <View />
```

Comment:
Tests are meant to cover all the cases and not just one

Reply:
Added more tests